plan/physical_plan_builder.go

```diff
@@ -498,7 +498,7 @@ func (ds *DataSource) forceToTableScan() PhysicalPlan {
 		Columns:     ds.Columns,
 		TableAsName: ds.TableAsName,
 		DBName:      ds.DBName,
-		Ranges:      ranger.FullIntNewRange(),
+		Ranges:      ranger.FullIntNewRange(nil),
```

Comment:
Why pass nil, the data source may have unsigned PK column.

Reply:
`forceToTableScan` only happens in IndexJoin. When execute index join, `Ranges` will be replaced with actual ranges. Don set it here?

Reply:
It depends too much on the assumption of its usage. The function itself doesn't imply its usage.
We can add a parameter to pass the pk column.